packages/core/test/unit/plugins/default/shorthandSyntax.test.js

```diff
@@ -0,0 +1,39 @@
+const cheerio = require('cheerio');
+const shorthandSyntaxPlugin = require('../../../../src/plugins/default/markbind-plugin-shorthandSyntax');
+
+test('processNode should convert shorthand syntax to proper Markbind syntax', () => {
+  const [spanNode] = cheerio.parseHTML('<panel>'
+    + '<span heading>Heading</span></panel>', true)[0].children;
+  shorthandSyntaxPlugin.processNode({}, spanNode);
+  expect(spanNode.type).toEqual('tag');
+  expect(spanNode.name).toEqual('span');
+  expect(spanNode.attribs.class).toEqual('card-title');
+  expect(spanNode.attribs.slot).toEqual('header');
+  expect(spanNode.children[0].data).toEqual('Heading');
+  expect(spanNode.children[0].type).toEqual('text');
+},
+);
+
+test('processNode should not convert span node without heading attribute', () => {
```

Comment:
I think maybe this can be included in the "improper syntax nodes" as well, but it's fine if we leave it here also. I leave it up to you :-)

Reply:
I decided to split the improper syntax test case into two so that it's clearer as to why it's improper. So I will keep this one separate as well. Thank you for the suggestion.